src/qibo/models/utils.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+from qibo import gates as maingates
+from qibo.gates import gates
+from qibo.models.circuit import Circuit
+
+
+class Loss_Utils(object):
```

Comment:
@shangtai since you moved the utils methods to a new file, it is simpler to remove this class completely and define them as standalone methods. Then you can import them directly in other modules, for example:
```py
from qibo.models.utils import cvar
```

Reply:
that would make the code cleaner :) thanks.